### DISCUSSION TITLE
Check if Pos is inside blocks

### DISCUSSION THREAD

**Discussion by Moulberry** [Minestom/Minestom#1022] [Help] [answered]
If I have a Pos and BoundingBox, how can I check if the BB is colliding with a block?

I am *not* trying to do entity movement, so BlockCollision#handlePhysics which requires a velocity isn't suitable

**Comment by OOP-778** [comment, marked as answer]
You can do so by getting the block shape then calling Shape#intersects(Pos, BoundingBox)
